### ISSUE TITLE
Export as PDF / Mobi / ePub

### ISSUE THREAD

**Issue by psiinon** [bryanbraun/franklin#9]
Thats one of my requirements for a documentation site - any chance of adding this?

**Comment by bryanbraun** [comment]
I agree, that would be useful. Ideally there would be some way to run the markdown through Pandoc, so you can build it into any format.

The last time [I looked into this](https://forum.middlemanapp.com/t/epub-generation-with-middleman/1523) I couldn't find any existing tooling. I can't commit the time to writing the integration from scratch so, for now, that feature is on hold.
